diff -uNr uboot-custom/scripts/dtc/dtc-parser.tab.c_shipped uboot-custom.patched/scripts/dtc/dtc-parser.tab.c_shipped
--- uboot-custom/scripts/dtc/dtc-parser.tab.c_shipped	2022-10-19 05:05:11.000000000 +0300
+++ uboot-custom.patched/scripts/dtc/dtc-parser.tab.c_shipped	2023-05-08 09:30:30.331204392 +0300
@@ -1202,7 +1202,7 @@
 /* The semantic value of the lookahead symbol.  */
 YYSTYPE yylval;
 /* Location data for the lookahead symbol.  */
-YYLTYPE yylloc
+extern YYLTYPE yylloc
 # if defined YYLTYPE_IS_TRIVIAL && YYLTYPE_IS_TRIVIAL
   = { 1, 1, 1, 1 }
 # endif
diff -uNr uboot-custom/scripts/Makefile.lib uboot-custom.patched/scripts/Makefile.lib
--- uboot-custom/scripts/Makefile.lib	2022-10-19 05:05:11.000000000 +0300
+++ uboot-custom.patched/scripts/Makefile.lib	2023-05-08 09:29:56.761646498 +0300
@@ -308,7 +308,7 @@
 # Modified for U-Boot
 # Bring in any U-Boot-specific include at the end of the file
 cmd_dtc = mkdir -p $(dir ${dtc-tmp}) ; \
-	(cat $<; $(if $(u_boot_dtsi),echo '\#include "$(u_boot_dtsi)"')) > $(pre-tmp); \
+	(cat $<; $(if $(u_boot_dtsi),echo '#include "$(u_boot_dtsi)"')) > $(pre-tmp); \
 	$(CPP) $(dtc_cpp_flags) -x assembler-with-cpp -o $(dtc-tmp) $(pre-tmp) ; \
 	$(DTC) -O dtb -o $@ -b 0 \
 		-i $(dir $<) $(DTC_FLAGS) \
